client/lib/wpcom-undocumented/lib/me.js

```diff
@@ -285,4 +285,12 @@ UndocumentedMe.prototype.deleteAccountRecoveryEmail = function( callback ) {
 	return this.wpcom.req.post( args, callback );
 };
 
+UndocumentedMe.prototype.removePurchase = function( purchaseId, fn ) {
```

Comment:
I wonder if we shouldn't name this function after the API endpoint, i.e. `deletePurchase()`.